types/aggregator.go

```diff
@@ -11,6 +11,7 @@ type Aggregator struct {
 	ID             string    `json:"id" yaml:"id"`
 	Token          string    `json:"token" yaml:"token"`
 	Name           string    `json:"name" yaml:"name"`
+	EnvironmentID  string    `json:"environment_id"`
```

Comment:
I propose passing a Environment string, validate is unique and save the id in the database.